src/App.tsx

```diff
@@ -15,6 +16,7 @@ function App() {
         <Header/>
         <Routes>
           <Route path="/login" element={<Login setUser={setUser}/>}/>
+          <Route path="/profile" element={<Profile/>}/>
```

Comment:
I would update the path to include an id to get specific accounts of people. I think its just like express where you can do ':id'.